core/components/organisms/textField/TextFieldWithTextarea.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import { Textarea, Label, HelpText, Text } from '@/index';
+import { BaseProps } from '@/utils/types';
+export interface TextFieldTextareaProps extends BaseProps {
```

Comment:
Instead of defining TextFieldTextareaProps please use the existing TextareaProps directly